File: notatka/app/src/main/java/com/locovna/notatka/model/Note.java
package com.locovna.notatka.model;

/**
 * Created by Darina Locovna on 1/17/17
 */

public class Note {
  private String mTitle;
  private String mBody;

  public Note(String title, String body) {
    mTitle = title;
    mBody = body;
  }

  public String getTitle() {
    return mTitle;
  }

  public void setTitle(String title) {
    mTitle = title;
  }

  public String getBody() {
    return mBody;
  }

  public void setBody(String body) {
    mBody = body;
  }
}

File: notatka/app/src/main/java/com/locovna/notatka/model/Note.kt
package com.locovna.notatka.model

import java.util.*

/**
 * A single note object
 *
 * @property id the name of this note.
 * @property title the title of this note.
 * @property body the text body of this note.
 * @constructor Creates an empty note with random id.
 */

class Note() {
    var id: UUID
    var title: String? = ""
    var body: String? = ""

    init {
        this.id = UUID.randomUUID()
    }

   constructor(noteId : UUID) : this() {
    this.id = noteId
   }

}
